src/main/java/org/jenkinsci/plugins/enhancedoldbuilddiscarder/EnhancedOldBuildDiscarder.java

```diff
@@ -308,6 +309,8 @@ private boolean shouldKeepRun(Run r, Run lsb, Run lstb) {
 	}
 
 	private boolean tooNew(Run r, Calendar cal) {
+		//Clock clockTest = Clock.systemDefaultZone();
```

Comment:
Can you get rid of the commented out code

Reply:
Good catch, done.